libimage/filters_test.go

```diff
@@ -100,6 +101,11 @@ func TestFilterReference(t *testing.T) {
 		listedImages, err := runtime.ListImages(ctx, listOptions)
 		require.NoError(t, err, "%v", test)
 		require.Len(t, listedImages, test.matches, "%s -> %v", test.filters, listedImages)
+		for _, image := range listedImages {
+			for _, name := range image.Names() {
+				require.Contains(t, test.names, name, "filters: %s ", test.filters)
```

Comment:
An `EqualValues` test, not just a subset, would be better.

Reply:
an in case the slice order is not deterministic ElementsMatch() is an option.

Reply:
Oops, yes. I don’t know how I found `EqualValues`.